OSDOCS#12145: Adds release notes for the unified perspectives (#93187)
openshift/openshift-docs: release_notes/ocp-4-19-release-notes.adoc

This release adds improvements related to the following components and concepts:

[id="ocp-release-notes-web-console_{context}"]

> === Web console

Author: What do you think of a special callout before the alphabetical list starts? Then having web console back alphabetically for the rest of the web console related release notes. 

I am open to any ideas! I just think this needs to be called out at the beginning somewhere since this is a really notable change.

> === Web console

Reviewer: Thanks for tagging me. I thought I had this fixed but something went wrong somewhere. I'll take a look now.

> === Web console

Reviewer: I'd prefer to stick with the alphabetic nature of names to not throw a customer off. They should see a logic in the naming pattern and one component name out of place might trigger a customer/doc user to raise a Jira bug. 

Will a banner or pop-up message be available on a 4.19 web console to also inform customers of the change?

> === Web console

Author: There is no planned banner as far as I know. I know they will have a blog post. I was playing around with it a bit and I have a few ideas. Could we maybe move "notable technical changes" to before "new features and enhancements" to call this out first? I just don't want such a notable change so far down in the RNs. WDYT? @dfitzmau @bergerhoffer @kalexand-rh

> === Web console

Reviewer 2: I know that there is a tour dialog that pops up when you first launch the 4.19 web console, so that's something at least.

I probably wouldn't recommend moving notable technical changes, just to keep things the same release-to-release. But you could still mention it there too if you wanted

But if you feel it's important enough to mention outside of the normal area, I wonder if you could sneak in a mention in it in "About this release". Like just a simple one-liner or something. Though I will say that that topic is getting a little long and maybe could do with subheadings for a few of the different areas :shrug:

> === Web console

Author: I'll look into this. Thanks for the feedback everyone! Going to get some reviews for now.

> Starting with {product-title} 4.19, the perspectives in the web console have unified to simplify navigation, reduce context switching, streamline tasks, and provide users with a more cohesive {product-title} experiance.

Reviewer: ```suggestion
Starting with {product-title} 4.19, the perspectives in the web console have unified to simplify navigation, reduce context switching, streamline tasks, and provide users with a more cohesive {product-title} experience.
```

[id="ocp-release-notes-web-console_{context}"]
=== Web console

Starting with {product-title} 4.19, the perspectives in the web console have unified to simplify navigation, reduce context switching, streamline tasks, and provide users with a more cohesive {product-title} experience.

> With this unified design, there is no longer a *Developer* perspective by default; however, _all_ {product-title} web console features are discoverable to all users. If you are not the cluster owner, you might need to request permission to certain features from the cluster owner. You can still enable the *Developer* perspective.

Reviewer 3: ```suggestion
With this unified design, there is no longer a *Developer* perspective in the default view; however, _all_ {product-title} web console features are discoverable to all users. If you are not the cluster owner, you might need to request permission for certain features from the cluster owner. You can still enable the *Developer* perspective.
```

> With this unified design, there is no longer a *Developer* perspective in the default view; however, _all_ {product-title} web console features are discoverable to all users. If you are not the cluster owner, you might need to request permission for certain features from the cluster owner. You can still enable the *Developer* perspective.

Reviewer 4: ```suggestion
With this unified design, there is no longer a *Developer* perspective in the default view, however; _all_ {product-title} web console features are discoverable to all users. If you are not the cluster owner, you might need to request permission for certain features from the cluster owner. You can still enable the *Developer* perspective.
```

Reviewer 4: You can still enable the developer perspective is confusing.

> With this unified design, there is no longer a *Developer* perspective in the default view; however, _all_ {product-title} web console features are discoverable to all users. If you are not the cluster owner, you might need to request permission for certain features from the cluster owner. You can still enable the *Developer* perspective.

Author: ```suggestion
With this unified design, there is no longer a *Developer* perspective in the default view; however, _all_ {product-title} web console features are discoverable to all users. If you are not the cluster owner, you might need to request permission for certain features from the cluster owner. The *Developer* perspective can still be manually enabled if you prefer.
```
What do you think about adding a few extra words here for context? There is no longer a dev perspective by default. @JoeAldinger

Reviewer 4: Yep, that's solid. I like it.

> The *Getting Started* pane in the web console provides resources such as, a tour of the console, information on setting up your cluster, a quick start for enabling the *Developer* perspective, and links to explore new features and capabilities.

Reviewer 4: ```suggestion
The *Getting Started* pane in the web console provides resources such as a tour of the console, information on setting up your cluster, a quick start for enabling the *Developer* perspective, and links to explore new features and capabilities.
```

> The *Getting Started* pane in the web console also provides resources such as, a tour of the console, information on setting up your cluster, and links to explore new features and capabilities.

Author: Moved back to web console section for now pending discussion for where to place it.

> The *Getting Started* pane in the web console also provides resources such as, a tour of the console, information on setting up your cluster, and links to explore new features and capabilities.

Reviewer 5: We have also added quick start to `Enable the developer perspective` and added it to `Getting Started` pane. Can we also add it here?

> //See xref:../web_console/web-console-overview#enabling-developer-perspective_web-console_web-console-overview for more information on enabling the *Developer* perspective.

Reviewer 4: Assuming this will go in when it works and isn't breaking builds.
